tests/scripts/quiet/cmake

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env bash
```

Comment:
Since the contents of `quiet/cmake` and `quiet/make` are the same except for the value of `NO_SILENCE`, is there any way to extract the common parts?

Reply:
That is much nicer